src/main/scala/scalaz/stream/Cause.scala

```diff
@@ -65,7 +65,7 @@ object Cause {
   sealed trait EarlyCause extends Cause
 
   object EarlyCause {
-    def apply[A](r:Throwable\/A):EarlyCause\/A =
+    def fromTaskResult[A](r:Throwable\/A):EarlyCause\/A =
```

Comment:
@djspiewak Yeah I found apply to be very confusing. On other hand I don't like the new name either, so if someone is better at english than me just please suggest name :-)